docs/versioned_docs/version-2.11.0/deprecated/ugcstor.md

```diff
@@ -0,0 +1,2496 @@
+   - metadata.labels.cstorpool.openebs.io/name
+   - metadata.labels.cstorpool.cstorpool.openebs.io/uid
+
+7. Create a CVR in the cStor pool identified in Step 5. Below is the template to create CVR on the identified cStor pool. The values in the template can be filled from CSP YAML related to identified cStor pool, and other CVRs of the volume.
+
+   ```
+   apiVersion: openebs.io/v1alpha1
+   kind: CStorVolumeReplica
+   metadata:
+    annotations:
+      cstorpool.openebs.io/hostname: <Kubernetes_node_name>
+      isRestoreVol: "false"
+      openebs.io/storage-class-ref: |
+        name: <storage_class_name>
+    finalizers: 
+    - cstorvolumereplica.openebs.io/finalizer
+    generation: 1
+    labels:
+      cstorpool.openebs.io/name: <csp_name>
+      cstorpool.openebs.io/uid: <csp_uid>
+      cstorvolume.openebs.io/name: <cstor_volume_name>
+      openebs.io/cas-template-name: <existing-cStor-volume-cas-template>
+      openebs.io/persistent-volume: <persistent_volume_name>
+      openebs.io/version: <openebs_version>
+    name: <cstor_volume_name>-<csp_name>
+    namespace: openebs
+   spec:
+    capacity: <initial_capacity>
+    targetIP: <target_service_ip>
+    replicaid: "<md5sum_of_pvc_uid_and_csp_uid>"
+   status:
+    phase: Recreate
+   versionDetails:
+     autoUpgrade: false
+     desired: <existing_cStor_volume_version>
+     status:
+       current: <existing_cStor_volume_version>
+   ```
+
+   - **<Kubernetes_node_name>**: Kubernetes node name where cStor pool exists and new CVR will be created on this Node. This can be obtained from step 6.
+
+   - **<storage_class_name>:** Storageclass name used to create the cStor volume. It is also available in any existing CVR. This can be obtained from step 3.
+
+   - **<csp_name>**:  Identified cStor pool name where new CVR will be created.  This can be obtained from step 6.
+
+   - **<csp_uid>**:  UID of identified cStor pool name where new CVR will be created.  This can be obtained from step 6.
+
+   - **<persistent_volume_name>**:  Kubernetes Persistent Volume name of the corresponding cStor volume. This can be obtained from step 3.
+
+   - **<openebs_version>**:  Version of OpenEBS on which this volume exist. This can be obtained from step 3.
+
+   - **<cstor_volume_name>**: Name of cStor volume. This can be get from step 3.
+
+   - **<cstor_volume_name>-<csp_name>**:  This is the new CVR name which is going to be created. This should be named as a combination of particular cStor volume name and identified cStor pool name. This can be get from step 3 and step 6.
+
+   - **<initial_capacity>**:  Capacity of the cStor volume. This can be get from step 3.
+
+   - **<target_service_ip>**:  Target IP of corresponding cStor volume. This can be got from step 3.
+
+   - **<md5sum_of_pvc_uid_and_csp_uid>**:  It is the unique value referred to as  `replicaid` in the whole cluster.  This can be generated by running the following command:
```

Comment:
```suggestion
   - **<md5sum_of_pvc_uid_and_csp_uid>**: It is the unique value referred to as `replicaid` in the whole cluster.  This can be generated by running the following command:
```
There are double spaces used in the docs. Please check those